bundlewatch.config.json

```diff
@@ -5,9 +5,9 @@
     { "path": "./examples/browser-rollup/dist/v4-size.js", "maxSize": "0.7 kB" },
     { "path": "./examples/browser-rollup/dist/v5-size.js", "maxSize": "1.5 kB" },
 
-    { "path": "./examples/browser-webpack/dist/v1-size.js", "maxSize": "1.3 kB" },
-    { "path": "./examples/browser-webpack/dist/v3-size.js", "maxSize": "2.5 kB" },
-    { "path": "./examples/browser-webpack/dist/v4-size.js", "maxSize": "1.0 kB" },
-    { "path": "./examples/browser-webpack/dist/v5-size.js", "maxSize": "1.9 kB" }
+    { "path": "./examples/browser-webpack/dist/v1-size.js", "maxSize": "1.0 kB" },
+    { "path": "./examples/browser-webpack/dist/v3-size.js", "maxSize": "2.1 kB" },
+    { "path": "./examples/browser-webpack/dist/v4-size.js", "maxSize": "0.7 kB" },
+    { "path": "./examples/browser-webpack/dist/v5-size.js", "maxSize": "1.5 kB" }
```

Comment:
Not strictly related to this PR but I noticed since I was working a lot with bundlewatch. It seems like WebPack got better with time and now matches the output size that Rollup gives us, so lowered these values to be the same for both.

Reply:
Nice!